src/test/java/io/appform/dropwizard/sharding/LegacyDBShardingBundleWithListenerTest.java

```diff
@@ -1,14 +1,8 @@
 package io.appform.dropwizard.sharding;
 
-import com.google.common.collect.Lists;
 import io.appform.dropwizard.sharding.config.ShardedHibernateFactory;
-import io.appform.dropwizard.sharding.dao.listeners.OrderItemTestListenerFactory;
-import io.appform.dropwizard.sharding.dao.listeners.TestListenerFactory;
 import io.appform.dropwizard.sharding.dao.testdata.entities.Order;
 import io.appform.dropwizard.sharding.dao.testdata.entities.OrderItem;
-import lombok.val;
-import org.junit.Assert;
-import org.junit.Test;
 
 public class LegacyDBShardingBundleWithListenerTest extends DBShardingBundleTestBase {
```

Comment:
BalancedDBShardingBundleWithListenerTest and this test class can be removed

Reply:
done